activation/poetdb.go

```diff
@@ -64,12 +73,16 @@ func (db *PoetDb) Validate(proof types.PoetProof, poetID []byte, roundID string,
 		return types.ProcessingError(fmt.Sprintf("invalid poet id %x", poetID))
 	}
 	root, err := calcRoot(proof.Members)
+	// we shouldn't care about poet proof with empty membership as it's not relevant.
+	if len(proof.Members) == 0 {
```

Comment:
can you double check we no longer has this log with this check?
```
{"L":"ERROR","T":"2022-04-15T08:23:53.242-0400","N":"35daa.poetListener ","M":"failed to store poet proof: failed to store poet proof for poetId 09146d5a32 round 4: exec: database: object exists","node_id":"35daab80af8a9960524a940cad5f8ab769d93afb483d618199da65b2ba5cf87a","module":"poetListener"}
```

Reply:
@countvonzero do you remember which systest was it?

Reply:
any systest. this is part of nodes setup.

Reply:
I've checked error fixed:

before (error is there) - http://35.223.10.224/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Loki%22,%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22test-icul%5C%22%7D%7C%20json%20%7C%3D%20%5C%22failed%20to%20store%20poet%20proof%5C%22%22%7D%5D

after (no error) - http://35.223.10.224/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Loki%22,%7B%22refId%22:%22A%22,%22expr%22:%22%7Bnamespace%3D%5C%22test-ndat%5C%22%7D%7C%20json%20%7C%3D%20%5C%22failed%20to%20store%20poet%20proof%5C%22%22%7D%5D

Reply:
cannot log in the provided link. can you provide screen shots instead? thanks

Reply:
I've sent you a password in Slack. Let me know if screenshots still needed.

Reply:
sure, here they are
<img width="1679" alt="Screenshot 2022-05-04 at 13 05 01" src="https://user-images.githubusercontent.com/178485/166652593-b37159b3-e96d-4f46-b1f7-4b6ddaf8dbae.png">
<img width="1679" alt="Screenshot 2022-05-04 at 13 05 12" src="https://user-images.githubusercontent.com/178485/166652605-1b7d82b7-e632-4bb4-9b64-dedf2e2c5810.png">